services/dis-promrulegroups-operator/dist/install.yaml

```diff
@@ -0,0 +1,259 @@
+apiVersion: v1
```

Comment:
was the dist folder pushed by mistake?

Reply:
Intentional, it's to show which changes I made to what's generated.